app/data/organizations.js

```diff
@@ -0,0 +1,24 @@
+export default {
+  coala: {
+    name: 'coala association e.V.',
+    trackers: [
+      {
+        type: 'github',
+        identifier: 'coala'
+      },
+      {
+        type: 'gitlab',
+        identifier: 'coala'
+      }
+    ]
+  },
+  "52-north-initiative-for-geospatial-open-source-software-gmbh": {
+    name: '52° North Initiative for Geospatial Open Source Software GmbH',
+    trackers: [
+      {
+        type: 'github',
+        identifier: '52North'
+      }
+    ]
+  }
```

Comment:
Move all the organizations from [app/organizations.js](https://github.com/coala/git-task-list/pull/33/files#diff-021eb88e611ed258074c798ed0e82eee) to here first, then the rest can be added in another PR.

Reply:
It cant, the other fetches from wikidata.